api/src/test/java/uk/gov/dhsc/htbhf/claimant/service/EligibilityAndEntitlementServiceTest.java

```diff
@@ -213,7 +236,7 @@ void shouldEvaluateClaimForGivenPaymentCycleWithEligibilityOverride(List<LocalDa
     @Test
     void shouldEvaluateClaimForGivenPaymentCycleWithExpiredEligibilityOverride() {
         //Given
-        Claim claim = aClaimWithEligibilityOverride(aConfirmedEligibilityWithNoChildrenOverriddenUntil(LocalDate.now()));
+        Claim claim = aClaimWithEligibilityOverride(aConfirmedEligibilityForPregnantTeenager(LocalDate.now()));
```

Comment:
Why was this changed? Seems like the original test should stay intact.

Reply:
ah i think it got changed during refactor, i will revert this.